K2Bridge/JsonConverters/BucketsCollectionConverter.cs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+
+namespace K2Bridge.JsonConverters
+{
+    /// <summary>
+    /// A converter able to serialize <see cref="Aggregations"/> to Elasticsearh response json format.
+    /// </summary>
+    internal class BucketsCollectionConverter
+    {
+        /*
```

Comment:
???

Reply:
Forgot to delete this class was made before JS's modifications.
Deleted